packages/core/src/sass/app/_common/components/wallet.scss

```diff
@@ -707,6 +707,44 @@ $wallet-bg-color: (
             ),
         ),
     ),
+    'xrp': (
```

Comment:
Do we use the `wallet-bg-color` mixin anywhere?

Reply:
yes @nijil-deriv 
Here we are using wallet-bg-color for wallet icon
![Screenshot 2024-07-22 at 15 37 58](https://github.com/user-attachments/assets/1d011659-f74f-4f33-9386-06fb64b55d82)